lib/install/management/virtual_app.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/vmware/vic/lib/install/data"
 	"github.com/vmware/vic/pkg/errors"
 	"github.com/vmware/vic/pkg/trace"
+  "github.com/vmware/vic/pkg/version"
```

Comment:
yup, i just realized my spacemacs doesn't have go layer installed and that was why. now it does :-)